src/main/java/blackjack/domain/CardDeck.java

```diff
@@ -0,0 +1,41 @@
+package blackjack.domain;
+
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Queue;
+
+public class CardDeck {
+
+    private Queue<Card> cards;

```

Comment:
```suggestion
    private final Queue<Card> cards;
```
변경되지 않기 때문에 final 로 선언해도 될 것 같아요.

Reply:
넵! 인텔리제이에서 경고도 띄웠었는데 못 봤었네요. 감사합니다.